packages/ember-glimmer/tests/integration/content-test.js

```diff
@@ -73,6 +73,8 @@ class DynamicContentTest extends RenderingTest {
   ['@test it can render a dynamic path']() {
     this.renderPath('message', { message: 'hello' });
 
+    expectNoDeprecation();
```

Comment:
Actually, the that test is testing that when you do `{{CaptializedPath}}`, *and* that you have a `CaptializedPath` property on the object, it does not emit a deprecation warning. Can you add this test back? (And remove this `expectNoDeprecation`)